Rocket.Chat/Resources/de.lproj/Localizable.strings

```diff
@@ -307,9 +307,9 @@
 "chat.messages.mentions.list.title" = "Erwähnungen";
 
 // Upload Banner
-"chat.banner.upload.uploading" = "Uploading %@"; // TODO
-"chat.banner.upload.error" = "Failed uploading %@"; // TODO
-"chat.banner.upload.retry" = "Try again"; // TODO
+"chat.banner.upload.uploading" = "Hochladen von %@";
+"chat.banner.upload.error" = "Hochladen von %@ ist fehlgeschlagen";
+"chat.banner.upload.retry" = "Versuchen Sie es noch einmal";
```

Comment:
"Erneut versuchen" would be sufficient if it's a button, I'll have to look it up :)

Reply:
@kaiiiiiiiii jep, it's an action, so I'd prefer the short version